class.mesh-ajax.php

```diff
@@ -121,20 +141,24 @@ function mesh_save_section() {
 	function mesh_choose_layout() {
 		check_ajax_referer( 'mesh_choose_layout_nonce', 'mesh_choose_layout_nonce' );
 
-		if ( ! $selected_template = sanitize_text_field( $_POST['mesh_section_layout'] ) ) {
-			$selected_template = 'mesh-columns-1.php';
+		if ( isset( $_POST['mesh_section_id'] ) ) { // Input var okay.
+			$section_id = absint( $_POST['mesh_section_id'] ); // Input var okay.
```

Comment:
This probably shouldn't be an `absint()` instead it should just be an `intval()`